src/main/java/ru/skypro/lessons/springboot/web/repository/EmployeeRepository.java

```diff
@@ -1,12 +1,22 @@
 package ru.skypro.lessons.springboot.web.repository;
 
-import ru.skypro.lessons.springboot.web.Employee;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+import ru.skypro.lessons.springboot.web.model.Employee;
+import ru.skypro.lessons.springboot.web.model.EmployeeFullInfo;
+import ru.skypro.lessons.springboot.web.model.Position;
 
 import java.util.List;
+@Repository
+public interface EmployeeRepository extends JpaRepository<Employee, Integer> {
 
-public interface EmployeeRepository {
-    List<Employee> sumSalariesEmployees();
-    List<Employee> minSalaryEmployee();
-    List<Employee> maxSalaryEmployee();
-    List<Employee> highAverageSalariesEmployees();
+    List<Employee> getEmployeeWithHighestSalary();
+    List<Employee> getEmployeesByPositionLike(Position position);
+
+    @Query("SELECT e.name , e.salary , p.name " +
```

Comment:
@Query("SELECT e.name , e.salary , p.name " +
             "FROM Employee e join fetch Position p " +
             "WHERE **e.id** = ?1 and e.position = p")